codegen.json

```diff
@@ -20,6 +20,7 @@
           "PingableServices": "./types/PingableServices#PingableServicesSource",
           "ProcessRecurrenceSuccess": "./types/ProcessRecurrenceSuccess#ProcessRecurrenceSuccessSource",
           "RemoveAuthIdentitySuccess": "./types/RemoveAuthIdentitySuccess#RemoveAuthIdentitySuccessSource",
+          "SetIsFreeMeetingTemplate": "./types/SetIsFreeMeetingTemplate#SetIsFreeMeetingTemplateSource",
```

Comment:
-1 This should be `"SetIsFreeMeetingTemplatePayload"` to match the GraphQL type.